src/pybind/mgr/cephadm/utils.py

```diff
@@ -21,14 +21,15 @@ def name_to_config_section(name: str) -> str:
     else:
         return 'mon'
 
+
 def name_to_auth_entity(daemon_type,  # type: str
                         daemon_id,    # type: str
-                        host = ""     # type  Optional[str] = ""
+                        host=''       # type  Optional[str] = ""
```

Comment:
convert to py3 style type checking?

Reply:
As mentioned in the PR description, this PR will only fix obvious issues. The whole code needs a complete refactoring which should happen in a separate PR. Additionally a lint infrastructure (e.g. using tox) should be established for the cephadm mgr module to prevent various Python issues in future.